org.adempiere.base/src/org/adempiere/base/Core.java

```diff
@@ -1061,4 +1062,18 @@ public static IReservationTracerFactory getReservationTracerFactory() {
 		
 		return DefaultReservationTracerFactory.getInstance();
 	}
+	
+	/**
+	 * Get tax lookup service
+	 * @return ITaxLookup service
+	 */
+	public static ITaxLookup getTaxLookup() {
+		String service = MSysConfig.getValue(MSysConfig.TAX_LOOKUP_SERVICE, Env.getAD_Client_ID(Env.getCtx()));
```

Comment:
My only suggestion would be to add explicitly here the default, like:
```
String service = MSysConfig.getValue(MSysConfig.TAX_LOOKUP_SERVICE, "org.adempiere.base.DefaultTaxLookup", Env.getAD_Client_ID(Env.getCtx()));
```

Just as a precaution, so this can keep working in case somebody delete or inactivate the SysConfig record.